src/components/timeline/TimelineCanvas.js

```diff
@@ -115,14 +104,18 @@ export default class TimelineCanvas<HoveredItem> extends PureComponent<
     const y: CssPixels = event.pageY - rect.top;
 
     const maybeHoveredItem = this.props.hitTest(x, y);
-    if (!hoveredItemsAreEqual(maybeHoveredItem, this.state.hoveredItem)) {
-      this.setState({ hoveredItem: maybeHoveredItem });
-    }
```

Comment:
This `if` block was useful to prevent renderings until we implemented tooltips, when we added the following `setState` call which made things awful.

So I'm removing it now in favor of calling `setState` in only specific cases, and a better `if` block in `componentDidUpdate`